### ISSUE TITLE
Don't detect wildcards in snakemake expand() method

### ISSUE THREAD

**Issue by iromeo** [JetBrains-Research/snakecharm#229]
Expand expands all variables, it is unlikely to get wilcard there (not sure about escaped curly braces, e.g. `expand("{{foo}} {bo}", bo=[1,2,3])`

E.g. false wildcard here:
![image](https://user-images.githubusercontent.com/72933/63619782-48799680-c5f8-11e9-9af0-77bd063c9358.png)

**Comment by iromeo** [comment]
P.S.: Probably we'd like to use some annotations to exclude some names from wildcards, e.g.:
```
# ignore_wildcard: not_a_wildcard1, not_a_wildcard2 
rule foo:
    input: some_method_with_injection("{not_a_wildcard1} {wildcard} {not_a_wildcard2}")
```

**Comment by iromeo** [comment]
wildcards not detected in expand: fixed in 5196ce879c945c042e35c46df312209777b44d88
Annotations is just suggestion, let's do not implement it now
